### DISCUSSION TITLE
rxjs.dev website down?

### DISCUSSION THREAD

**Discussion by codingjlu** [ReactiveX/rxjs#6880] [Report issues other than bug]
I just tried to visit https://rxjs.dev and it gave me a Firebase error saying `index.html` wasn't existent.
![image](https://user-images.githubusercontent.com/78052093/157563564-14ebc81e-af8d-4638-bb1e-4881a0f64d33.png)
Did someone delete on accident or is it just my problem?

**Comment by jakovljevic-mladen** [comment]
Thanks for reporting. The site should be working fine now. Sorry for the inconvenience.
